### ISSUE TITLE
[Profile-Admin] Controller to get the question addressed to the user o department.  

### ISSUE THREAD

**Issue by GeorginaFraire** [wizeline/wize-q-remix#106]
Create a controller to get the question addressed to a user o a department. 
These are the field to use in the `where` clause: 
For the user: `assigned_to_employee_id`
For the department : `assigned_department`

**Comment by albertcastaned** [comment]
The general listQuestions controller has a department filter that we can use. I think we could also implmenet a similar filter for the employee id, that way we can share the same list questions functionality for the profile page. What do you think? @GinaFraMi

**Comment by GeorginaFraire** [comment]
Yes, I was thinking that we can have the same filter that we have in the homepage, but instead of get all questions, get just the ones for the department the admin/user belongs.
- Status
- Date Range
- Location

**Comment by albertcastaned** [comment]
@GeorginaFraire Actually, couldnt we use this to get the users questions created / commented ? (except for anonymous for the moment) https://github.com/wizeline/wize-q-remix/blob/master/app/controllers/profile/questionsCreated.js

**Comment by GeorginaFraire** [comment]
For the department, we can use the list controller, but for the user, we need to add a new filter to find what questions have been asked for the admin with the field  `assigned_to_employee_id`

**Comment by albertcastaned** [comment]
Will we still need this controller? For the profile view? Or is this feature no longer a priority? @GeorginaFraire

**Comment by GeorginaFraire** [comment]
Yes, @albertcastaned. We still need this for the profile.
